layouts/default.vue

```diff
@@ -1,5 +1,18 @@
 <template>
-  <div>
-    <Nuxt />
+  <div class="container mx-auto m-0 p-2 h-screen max-w-4xl">
+    <div class="flex m-4 justify-around">
+      <NuxtLink :to="`/`">
+        <img src="~/static/eucp_logo.png" alt="EUCP Logo">
+      </NuxtLink>
+      <div class="flex m-4 justify-around items-center">
+        <NuxtLink :to="`/`">
+          <p class="text-sm text-gray-400">
+            Home
+          </p>
+        </NuxtLink>
+          <input class="w-32 h-8 focus:shadow-outline rounded text-sm px-2 pl-4" type="search" placeholder="Search...">
```

Comment:
Nice idea to add a search box. Did you already have some ideas as to how to actually make it work?

Reply:
There are a few examples for search [here](https://tailwindcomponents.com/search?query=search). I am not sure whether we want to have a search box. if so, I will explore how to make it work.